build.gradle.kts

```diff
@@ -105,9 +121,15 @@ tasks {
         shouldRunAfter(test)
     }
 
+    jacocoTestReport {
+        reports {
+            xml.isEnabled = true
+        }
+    }
+
     check {
         // to find Javadoc errors early, let "javadoc" task run during "check"
-        dependsOn(javadoc, validateYaml)
+        dependsOn(javadoc, validateYaml, jacocoTestReport, sonarqube)
```

Comment:
this change is controversial, for me sonarqube is part of the check, but, we can also think of removing it.